frontend/actions/index.js

```diff
@@ -72,7 +77,7 @@ export const fetchRechargeCart = () => (dispatch, getState) => {
   const state = getState();
   const rechargeCartState = getRechargeCartState(state);
 
-  if (rechargeCartState.isFetching) {
+  if (rechargeCartState.isFetching || rechargeCartState.isPaused) {
```

Comment:
should we worry about restarting the fetch cart after it is unpaused?

Reply:
We should not. Updated name to reflect that it is blocking instead of pausing